### PR TITLE
Feature/twitter space cleanup

### PR DESCRIPTION
**Pull request by amaredeus**
## 2.1.0-SNAPSHOT (2021-11-22)

1. Add Privacy Policy and Terms & Conditions to /docs
2. Prepare twitter spaces command
   - end poap events cleanly on startup or restart of instance
   - remove period from poap link

### CODE REVIEW

File: src/app/service/poap/StartPOAP.ts [hunk 1]
```diff
@@ -282,7 +286,7 @@ const startTwitterPOAPFlow = async (ctx: CommandContext, guildMember: GuildMembe
 	
 	if (activeSettings != null) {
 		Log.debug('unable to start twitter event due to active event');
-		await guildMember.send('Looks like you are already tracking your twitter space!');
+		await guildMember.send('Looks like you have already started you POAP twitter space!');
```
Comment (nonsensecodes) [inline src/app/service/poap/StartPOAP.ts:289]: '... your POAP twitter space!'

File: src/app/service/poap/StartPOAP.ts [hunk 2]
```diff
@@ -350,5 +354,5 @@ const startTwitterPOAPFlow = async (ctx: CommandContext, guildMember: GuildMembe
 	if (result.ok != 1) {
 		await guildMember.send('One more thing, there was a problem setting counting you as a participants.. You just might miss the POAP, sorry.');
```
Comment (nonsensecodes) [inline src/app/service/poap/StartPOAP.ts:355]: '... there's a problem. Could be you. Could be me. But looks like you weren't setup properly to receive a POAP. Reach out the the event coordinator for your badge.'

File: src/app/service/poap/StartPOAP.ts [hunk 3]
```diff
@@ -350,5 +354,5 @@ const startTwitterPOAPFlow = async (ctx: CommandContext, guildMember: GuildMembe
...
 	}
-	Log.debug('POAP Twitter spaces event tracking message sent');
+	Log.debug('POAP Twitter spaces event event start message sent');
```
Comment (nonsensecodes) [inline src/app/service/poap/StartPOAP.ts:357]: Did you purposely put two "events" here? I know it's only for logging, but just want to be sure this was intentional.

**Review by nonsensecodes — CHANGES_REQUESTED** [review]
Just a few comments in one file.

**Comment by nonsensecodes** [comment]
Looks like the change log needs to be updated too.

File: src/app/service/poap/StartPOAP.ts (hunk 3 above)
Comment (amaredeus) [reply src/app/service/poap/StartPOAP.ts:357]: dam I'm too young to be stroking lol

**Review by nonsensecodes — APPROVED** [review]
Looks good! Thank you, @amaredeus

**Pull request merged**